spec/services/shopify_import/creators/custom_collection_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+RSpec.describe ShopifyImport::Creators::CustomCollection, type: :service do
+  subject { described_class.new(shopify_data_feed) }
+
+  before { ShopifyAPI::Base.site = 'https://api_key:passowrd@shop_domain.myshopify.com/admin' }
+
+  describe '#save!', vcr: { cassette_name: 'shopify/base_custom_collection' } do
+    let(:shopify_custom_collection) { ShopifyAPI::CustomCollection.find(388_567_107) }
```

Comment:
@vfonic I added this because shopify collection does not have products as attribute. So i prefare to create real data model here.